app/lib/view/moodle.dart

```diff
@@ -0,0 +1,233 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:webview_inapp/flutter_inappwebview.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../client/client.dart';
+
+class MoodleWebView extends StatefulWidget {
+  const MoodleWebView({super.key});
+
+  @override
+  State<MoodleWebView> createState() => _MoodleWebViewState();
+}
+
+class _MoodleWebViewState extends State<MoodleWebView> {
+  ValueNotifier<bool> canGoBack = ValueNotifier(false);
+  ValueNotifier<bool> canGoForward = ValueNotifier(false);
+  ValueNotifier<int> progressIndicator = ValueNotifier(0);
+  ValueNotifier<bool> hideWebView = ValueNotifier(false);
+
+  InAppWebViewController? webViewController;
+  PullToRefreshController? pullToRefreshController;
+
+  @override
+  void initState() {
+    super.initState();
+
+    String sessionToken = "";
+    client.jar
+        .loadForRequest(Uri.parse("https://start.schulportal.hessen.de"))
+        .then((cookies) {
+      sessionToken = cookies[1].value;
```

Comment:
Is a hard coded index of 1 unsafe here?

Reply:
should be safe (at least on my android chrome and desktop firefox) but I will code a safer option